tests/diff_test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-. ./src/diff.sh --source-only
-. ./tests/utils --source-only
+include "./src/diff.sh"
+include "./tests/utils"
```

Comment:
For your next version, prefer a single quote here.
(please, review all of these quotes in your series).